azure-pipelines.yml

```diff
@@ -226,6 +226,18 @@ stages:
       helixQueueName: $(HelixWindowsQueueName)
       helixApiAccessToken: $(HelixApiAccessToken)
 
+  # Uncomment before merge
```

Comment:
Once the PR gets full sign off I will push a commit that moves this to CI only. Want to keep it running on PRs so I can validate any changes I make are indeed passing the es-ES legs.